spec/models/media/object_spec.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
-# rubocop:disable Layout/LineLength
+# rubocop:disable Layout/MultilineHashBraceLayout
+# rubocop:disable Layout/HashAlignment
+# rubocop:disable Style/WordArray
```

Comment:
The LineLength thing actually gets added automatically for these docs, so these may be destroyed next time we run a schema migration. For safety let's put them lower like above `describe Media::Object`